src/dodal/devices/electron_analyser/vgscienta/region.py

```diff
@@ -32,23 +32,25 @@ class VGScientaRegion(
     high_energy: float = 10.0
     step_time: float = 1.0
     energy_step: float = Field(default=200.0)
+    centre_energy: float = Field(alias="fix_energy", default=9)
     # Specific to this class
     id: str = Field(default=str(uuid.uuid4()), alias="region_id")
-    fix_energy: float = 9.0
     total_steps: float = 13.0
     total_time: float = 13.0
-    first_x_channel: int = 1
-    last_x_channel: int = 1000
-    first_y_channel: int = 101
-    last_y_channel: int = 800
+    min_x: int = Field(alias="first_x_channel", default=1)
+    sensor_max_size_x: int = Field(alias="last_x_channel", default=1000)
+    min_y: int = Field(alias="first_y_channel", default=101)
+    sensor_max_size_y: int = Field(alias="last_y_channel", default=800)
```

Comment:
Could: Assuming you have files that still use `first_x_channel` etc. in production I would add some tests that confirm these alias works. You can just have one catch-all "test_backwards_compatibility_with_old_region_format"

Reply:
Ah, I missed this cos I was just searching the codebase for `first_x_channel` without realising you had magic to convert `firstXChannel` to this